reesegreen2014/dog-info-hub
# Feature: Owner Resources Page

#### What does this PR do?
- This PR introduces a new "Owner Resources" page that displays cards linking to useful articles for pet owners.
#### Where should the reviewer start?
- Review the `OwnerResources.js` and `OwnerResources.css` files in the components/OwnerResources directory.
#### How should this be manually tested?
1. Navigate to the Owner Resources Page:
- [ ] Click on the "Owner Resources" link in the header.
- [ ] Verify that the page displays cards with images, titles, descriptions, and links to external articles.
2. Check Links:
- [ ] Click on each card's "Read More" link to ensure it opens the corresponding article in a new tab.
#### Any background context you want to provide?
- This new page aims to provide valuable resources to pet owners.
#### What are the relevant tickets (if any)?
#16 is linked to this PR and will be closed upon successful merge of this PR. 
#### Screenshots (if appropriate)
![image](https://github.com/user-attachments/assets/586e27dd-5534-4a3f-978f-d33714cd35e1)
